.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- This form is for bug reports and feature requests ONLY! 
+
+If you're looking for help check [Vault's Google Group](https://groups.google.com/forum/#!forum/vault-tool).
+-->
+
+**Type of report:**
+
+[ ] Bug
+[ ] Feature request
+
+**Environment:**
+
+* **Version**:
+* **Platform**:
+* **Subsystem**:
```

Comment:
I borrowed this from nodejs' template, but I was thinking of that -- i.e. backend, cli, core, etc.